### ISSUE TITLE
YaST not launching / missing polkit policy endless loop 

### ISSUE THREAD

**Issue by Albireo67** [openSUSE/openSUSEway#123]
I am confused by the behavior of YaST, and unsure whether my installation is broken or missing anything. Let me explain:

- If I launch `yast` from `wofi`, an `xdg-su` window opens, and once password is entered, it goes into an endless loop of missing polkit policy errors (!?)
- If I launch `yast` from a terminal, prepended with `sudo`,  it opens the ncurses TUI and works just fine. Theme is almost unreadable, but it works.
- If I try with another launcher, say `fuzzel`, no `xdg-su`window opens, so no password can be entered, though a YaST GUI launches showing only non-privileged components (e.g. Help), so it's useless.

- I temporarily fixed this issue by editing `/etc/sysconfig/yast2` and changing these two lines:

   `WANTED_GUI="ncurses"`

   `Y2NCURSES_COLOR_THEME="inverted"`  # Optional. This one  is for a more readable theme

- After that, I can successfully launch `yast` from `wofi` or `fuzzel`, and  in both cases an `xdg-open` window is launched asking for a password and it gracefully opens the ncurses TUI in all cases with no issues. 
- Launching via terminal command: no issues, just like before.
 
What's the expected behavior on a fresh installation? No GUI and always ncurses TUI ?
Or is my system simply missing some files, hence lacking some features ?

**Comment by FilippoBonazziSUSE** [comment]
Sorry, I don't use YaST so I don't know. I don't know what package would provide a YaST GUI, and we probably are not including that in openSUSEway because it has not been of interest of any contributor. Likewise, I don't know what `.desktop` entries are provided by YaST which would be picked up by a launcher, if they are DE-specific or how they are supposed to work.
Maybe someone else that uses YaST can advise on the best way to use it in openSUSEway (I suspect it will be the ncurses interface).

**Comment by ucihariskoding** [comment]
yeah, i found this problems too. after i saw the fix from @Albireo67 , i tried to change "WANTED_GUI=" into "qt" instead of ncurses and it worked fine.

**Comment by FilippoBonazziSUSE** [comment]
I am not sure we want to modify system-wide YaST configuration in openSUSEway. Also not sure how to ship an additional YaST config that would not break its normal usage on another desktop.
